packages/app-mobile/components/screens/ConfigScreen.tsx

```diff
@@ -114,11 +114,23 @@ class ConfigScreenComponent extends BaseScreenComponent {
 			const logItemCsv = service.csvCreate(logItemRows);
 
 			const itemListCsv = await service.basicItemList({ format: 'csv' });
-			const filePath = `${RNFS.ExternalDirectoryPath}/syncReport-${new Date().getTime()}.txt`;
 
-			const finalText = [logItemCsv, itemListCsv].join('\n================================================================================\n');
+			let filePath;
+			if (Platform.OS === 'android' && Platform.Version > 28) {
```

Comment:
All these platform checks need to be part of shim.fsDriver(). Maybe create an `async getExternalDirectoryPath()` method

Reply:
okay, I'll give it a go, I think I got the idea